artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/handler/ProtonHandler.java

```diff
@@ -443,7 +445,9 @@ private void dispatch() {
             inDispatch = true;
             while ((ev = collector.peek()) != null) {
                for (EventHandler h : handlers) {
-                  if (log.isTraceEnabled()) {
+                  if (PN_TRACE_FRM) {
+                     log.info("Handling " + ev + " towards " + h);
```

Comment:
This also doesn't feel like it should be tied to having proton trace its frames to me. Given the existing trace logging I wouldn't do this.

Reply:
It is often what I need when debugging AMQP.. I wanted to have an easy easy to turn on and debug at console without having to turn trace for everything (It's a bit diefficult to do individually).